docker-compose.yml

```diff
@@ -3,11 +3,8 @@ version: '3.9'
 services:
   app-elasticsearch:
     container_name: stac-fastapi-es
-    image: stac-utils/stac-fastapi-es
+    image: ghcr.io/stac-utils/stac-fastapi-es:latest
```

Comment:
Can you revert this to pulling from the local code like it was doing before (`stac-utils/stac-fastapi-es`)? The docker-compose file is used for local development, so we want it to build an image from the local code.